burger-app/src/components/Main/index.js

```diff
@@ -1,74 +1,108 @@
+import React from "react";
 import styled from "styled-components";
+import Prices from "./Prices";
+import Controls from "./Controls";
+import Burger from "./Burger";
+import axios from "axios";
 
-function Main() {
-    return (
-        <MainWrapperStyled>
-            <BlogWrapperStyled>
-                <BlogHeaderStyled>Blog</BlogHeaderStyled>
-                <BlogItemStyled></BlogItemStyled>
-                <BlogItemStyled></BlogItemStyled>
-                <BlogItemStyled></BlogItemStyled>
-                <BlogItemStyled></BlogItemStyled>
-            </BlogWrapperStyled>
-            <NewsWrapperStyled>
-                <NewsHeaderStyled>News</NewsHeaderStyled>
-                <NewsItemStyled></NewsItemStyled>
-                <NewsItemStyled></NewsItemStyled>
-                <NewsItemStyled></NewsItemStyled>
-            </NewsWrapperStyled>
-        </MainWrapperStyled>
-    )
-};
+const baseUrl = "https://burger-api-xcwp.onrender.com/ingredients";
```

Comment:
The base URL should be as a base. `/ingredients` will change to `/orders`, `/contacts` etc.

```suggestion
const baseUrl = "https://burger-api-xcwp.onrender.com/";
```